bin/reth/src/chain/mod.rs

```diff
@@ -0,0 +1,53 @@
+use clap::Parser;
+use eyre::WrapErr;
+use reth_primitives::Block;
+use std::path::PathBuf;
+
+#[derive(Debug, Parser)]
+pub struct ImportCommand {
+    /// The file to import.
+    path: PathBuf,
+}
+
+impl ImportCommand {
+    pub async fn execute(&self) -> eyre::Result<()> {
+        // read file
+        let mut contents = &std::fs::read(&self.path).wrap_err("Could not open file")?[..];
```

Comment:
Wrap in a BufReader to make faster, and just open the file instead of reading in all the contents to avoid memory usage blowing up bc of loading an entire large file